webapp/src/styles/_components.scss

```diff
@@ -0,0 +1,13 @@
+.project-details-unlink-button {
+    max-width: 42px;
+    height: 34px;
+    margin-right: 16px;
+    display: block;
+    margin-left: auto;
+}
+
+.unlink-button {
```

Comment:
we have kept it consistent with Mattermost as it uses kebab-case but its a good point, please create a ticket on MI board and assign to @sooraj-shukla we will refactor the code with such standard practises together when we would be having an UI library ready

Reply:
@avas27JTG I noticed that mattermost also uses BEM and that's why most of the components that we've created are already following BEM.
We can name classes like this(Eg. `unlink-button`, `text-center` etc.) even if we're following BEM. Basically util classes or classes that we can use at multiple places in our app can follow kebab case naming conventions because they aren't related to any parent or something.

So, if this issue don't exist with other PRs in azure then we can fix this here only without creating a seperate ticket.

Reply:
Yes this is a util kind of class and BEM is not really needed here we can leave it
but still I or some other dev might have created classes in some components as well so it would be good to be on a safer side to have a backlog ticket for this

Reply:
sure!